add-ons/che/README.md

```diff
@@ -0,0 +1,68 @@
+# Eclipse Che Addon
+
+This addon create Eclipse Che templates, image streams and a project running Che in Minishift. In short this helps in setting up Eclipse Che 
+on minishift inline with [Deploy Che on Minishift](https://www.eclipse.org/che/docs/setup/openshift/index.html#deploy-che-on-minishift)
```

Comment:
`on Minishift inline with`.

`Minishift` as brand name and `minishift` for CLI purpose.